src/PaymentResolver/PaymentMethodsResolver.php

```diff
@@ -9,35 +9,27 @@
 use Sylius\Component\Core\Model\PaymentInterface;
 use Sylius\Component\Core\Repository\PaymentMethodRepositoryInterface;
 use Sylius\Component\Payment\Model\PaymentInterface as BasePaymentInterface;
+use Sylius\Component\Payment\Model\PaymentMethodInterface;
 use Sylius\Component\Payment\Resolver\PaymentMethodsResolverInterface;
+use ThreeBRS\SyliusPaymentRestrictionPlugin\Model\PaymentMethodRestrictionInterface;
 use ThreeBRS\SyliusPaymentRestrictionPlugin\Model\ThreeBRSSyliusResolvePaymentMethodForOrder;
 use Webmozart\Assert\Assert;
 
 class PaymentMethodsResolver implements PaymentMethodsResolverInterface
 {
-    /** @var PaymentMethodRepositoryInterface */
-    private $paymentMethodRepository;
-
-    /** @var ThreeBRSSyliusResolvePaymentMethodForOrder */
-    private $paymentOrderResolver;
-
-    public function __construct(
-        PaymentMethodRepositoryInterface $paymentMethodRepository,
-        ThreeBRSSyliusResolvePaymentMethodForOrder $paymentOrderResolver
-    ) {
-        $this->paymentMethodRepository = $paymentMethodRepository;
-        $this->paymentOrderResolver = $paymentOrderResolver;
+    public function __construct(private PaymentMethodRepositoryInterface $paymentMethodRepository, private ThreeBRSSyliusResolvePaymentMethodForOrder $paymentOrderResolver)
```

Comment:
moc dlouhý řádek, nechal bych to jak to bylo

Reply:
To udělal Rector při převodu kódu na PHP 8.0

Já bych to zas nechal moderně.

Upravil jsem to, aby to nebylo ošklivě roztáhnuté
```php
    public function __construct(
        private PaymentMethodRepositoryInterface $paymentMethodRepository,
        private ThreeBRSSyliusResolvePaymentMethodForOrder $paymentOrderResolver,
    ) {
    }
```